Makefile

```diff
@@ -33,6 +33,12 @@ clean_remote:
 populatedb:
 	exec docker exec -it lantern-back-end_endpoint_manager_1 /etc/lantern/populatedb.sh
 
+backup_database:
+	docker exec lantern-back-end_postgres_1 pg_dump -Fc -U lantern -d lantern > lantern_backup_`date +%Y%m%d%H%M%S`.sql
```

Comment:
Super  minor comment but can you add an `echo` statement  that prints out that the database was backed up to <file name> ... and same for the restore

Reply:
Just made this change, these commands now only print out "Database was backed up to <filename>" and "Database was restored to <filename>"